assets/runtime/env-defaults

```diff
@@ -537,6 +537,28 @@ OAUTH2_GENERIC_AUTHORIZE_PARAMS_SCOPE=${OAUTH2_GENERIC_AUTHORIZE_PARAMS_SCOPE:-}
 OAUTH2_GENERIC_LABEL=${OAUTH2_GENERIC_LABEL:-}
 OAUTH2_GENERIC_NAME=${OAUTH2_GENERIC_NAME:-}
 
+### OpenID Connect
+OAUTH_OIDC_LABEL=${OAUTH_OIDC_LABEL:-'OpenID Connect'}
+OAUTH_OIDC_ICON=${OAUTH_OIDC_ICON:-}
+OAUTH_OIDC_SCOPE=${OAUTH_OIDC_SCOPE:-"['openid','profile','email']"}
+OAUTH_OIDC_RESPONSE_TYPE=${OAUTH_OIDC_RESPONSE_TYPE:-'code'}
+OAUTH_OIDC_ISSUER=${OAUTH_OIDC_ISSUER:-}
+OAUTH_OIDC_DISCOVERY=${OAUTH_OIDC_DISCOVERY:-true}
+OAUTH_OIDC_CLIENT_AUTH_METHOD=${OAUTH_OIDC_CLIENT_AUTH_METHOD:-'basic'}
```

Comment:
In <https://docs.gitlab.com/ee/administration/auth/oidc.html> the default value is `'query'`. Does tis matter?

Reply:
In https://docs.gitlab.com/ee/administration/auth/oidc.html under step 4, supported values for `client_auth_method` are listed: `basic`, `jwt_bearer`, `mtls`, *Any other value posts the client ID and secret in the request body*. *If not specified, this value defaults to `basic`.* However `query` is written in the examples. Finally, I decided to use the `basic` value.